### PR TITLE
Update BlueWalletRelease.entitlements

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

File: ios/BlueWallet/BlueWalletRelease.entitlements [hunk 1]
```diff
@@ -5,13 +5,17 @@
 	<key>aps-environment</key>
 	<string>development</string>
 	<key>com.apple.developer.icloud-container-identifiers</key>
-	<array/>
+	<array>
+		<string>iCloud.io.bluewallet.bluewallet</string>
```
Comment (Overtorment) [inline ios/BlueWallet/BlueWalletRelease.entitlements:9]: is that okay that `C` is capitalized here?

File: ios/BlueWallet/BlueWalletRelease.entitlements [hunk 2]
```diff
@@ -5,13 +5,17 @@
...
+	</array>
 	<key>com.apple.developer.icloud-services</key>
 	<array>
 		<string>CloudDocuments</string>
 	</array>
 	<key>com.apple.developer.ubiquity-container-identifiers</key>
-	<array/>
+	<array>
+		<string>iCloud.io.bluewallet.bluewallet</string>
```
Comment (Overtorment) [inline ios/BlueWallet/BlueWalletRelease.entitlements:17]: and here

**Review by Overtorment — APPROVED** [review]

File: ios/BlueWallet/BlueWalletRelease.entitlements (hunk 2 above)
Comment (marcosrdz) [reply ios/BlueWallet/BlueWalletRelease.entitlements:17]: yup. default by xcode

File: ios/BlueWallet/BlueWalletRelease.entitlements (hunk 1 above)
Comment (marcosrdz) [reply ios/BlueWallet/BlueWalletRelease.entitlements:9]: yup. default by xcode

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/n4uetrw0brgfaw4p5th2b9vg94

[download apk](http://filestorage.bluewallet.io:1488/1610741440.apk)

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

**Pull request merged**